examples/src/fibonacci/fib2/mod.rs

```diff
@@ -98,10 +98,13 @@ impl Example for FibExample {
     }
 
     fn verify(&self, proof: StarkProof) -> Result<(), VerifierError> {
-        winterfell::verify::<FibAir>(proof, self.result)
+        winterfell::verify::<FibAir, <FibProver as Prover>::HashFn>(proof, self.result)
```

Comment:
Could we define `Blake3` as a type at the top of this module and then use it both here and in the `prover.rs`? It could look something like this:
```Rust
type Blake3 = winterfell::crypto::hashers::Blake3_256<BaseElement>;
```

Reply:
We might not need to do this as we are making it accept any generic hash function now.